flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -461,14 +461,20 @@ public RestartStrategies.RestartStrategyConfiguration getRestartStrategy() {
         }
     }
 
+    /**
+     * TODO: this shouldn't exist and shouldn't pollute public API. Tests should change this via
```

Comment:
This isn't too difficult to fix though, is it? Can't we modify the config via the TableEnvironment in the affected tests?

Reply:
Ahhh, I've tried to fix this but I couldn't do it easily. One place was easy, but the other three were not :(

Reply:
Per offline discussion we decided to leave it as it is, due to some quirks and difficulties in fixing in `TableTestUtil`